packages/js/core/src/__tests__/resolveUri.spec.ts

```diff
@@ -231,10 +225,10 @@ describe("resolveUri", () => {
     },
   ];
 
-  const apis: Record<string, PluginModules> = {
-    "w3://ens/ens": ensApi,
-    "w3://ens/ipfs": ipfsApi,
-    "w3://ens/my-plugin": pluginApi,
+  const apis: Record<string, PluginModule> = {
+    "w3://ens/ens": ensApi as unknown as PluginModule,
+    "w3://ens/ipfs": ipfsApi as unknown as PluginModule,
+    "w3://ens/my-plugin": pluginApi as unknown as PluginModule,
```

Comment:
If we need to do this kind of casting, wouldn't it make sense to have an interface that abstracts part of what currently is in the abstract class?

Reply:
this is indeed a dirty hack, and should be cleaned up in the future, but nothing "wrong" with it right now